File: PRAK302-2310817210026-MUHAMMADPUTRAAZKYALFANI.c
#include <stdio.h>

int main() {
    int nilai;
    char nilai_huruf;

    printf("Masukkan nilai: ");
    scanf("%d", &nilai);

    if (nilai >= 90) {
        nilai_huruf = 'A';
    } else if (nilai >= 70) {
        nilai_huruf = 'B';
    } else if (nilai >= 60) {
        nilai_huruf = 'C';
    } else if (nilai >= 50) {
        nilai_huruf = 'D';
    } else {
        nilai_huruf = 'E';
    }

   
    printf("Nilai huruf: %c\n", nilai_huruf);

    return 0;
}

File: PRAK302-2310817210026-MUHAMMADPUTRAAZKYALFANI.py
def konversi_nilai(nilai):
    if nilai >= 90:
        return "A"
    elif nilai >= 70:
        return "B"
    elif nilai >= 60:
        return "C"
    elif nilai >= 50:
        return "D"
    else:
        return "E"

nilai = int(input("Masukkan nilai: "))

hasil_konversi = konversi_nilai(nilai)

print("Hasil konversi nilai:", hasil_konversi)
